parsers/go.c

```diff
@@ -136,15 +136,13 @@ static tokenInfo *newToken (void)
 	return token;
 }
 
-static tokenInfo *copyToken (tokenInfo *other)
+static void copyToken (tokenInfo *const dest, const tokenInfo *const other)
 {
-	tokenInfo *const token = xMalloc (1, tokenInfo);
```

Comment:
Why wouldn't it be just as easy to use objpool here inside `copyToken()`?
I don't mean the change is bad, but I don't really see why it's that useful.